### DISCUSSION TITLE
Profile banner for organisations

### DISCUSSION THREAD

**Discussion by NitinIsFit** [community/community#35767] [Other Feature Feedback, Questions, & Ideas] [answered]
Add a function for organisations to add a profile banner.

**Comment by jfullstackdev** [comment]
you mean like this ? 

![image](https://user-images.githubusercontent.com/47092464/195122973-8305bb44-179c-4308-bb2f-9f762d88dbe6.png)

check this out 

https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile#adding-a-public-organization-profile-readme

**Reply by NitinIsFit** [reply]
naah

**Reply by NitinIsFit** [reply]
like this
![Zombodroid_13102022104029](https://user-images.githubusercontent.com/94877880/195507145-28e0d538-2f1b-422f-9e76-4765a7b3f2ae.jpg)

**Comment by MeesJ** [comment, marked as answer]
+1, this would be amazing, even for personal GitHub profiles.
